File: src/main/java/com/example/kyudo_app/infrastructure/external/AssignBelongingGroupRepository.java
package com.example.kyudo_app.infrastructure.external;

import com.example.kyudo_app.domain.model.BelongingGroup;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface AssignBelongingGroupRepository extends JpaRepository<BelongingGroup, Long> {
    Optional<BelongingGroup> findById(Long groupId);
}

File: src/main/kotlin/com/example/kyudo_app/infrastructure/external/AssignBelongingGroupRepository.kt
package com.example.kyudo_app.infrastructure.external

import com.example.kyudo_app.domain.model.BelongingGroup
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface AssignBelongingGroupRepository : JpaRepository<BelongingGroup?, Long?> {
    override fun findById(groupId: Long?): Optional<BelongingGroup?>
}
